### ISSUE TITLE
Feature Request: Shipyard Name-Field Paste

### ISSUE THREAD

**Issue by VaelophisNyx** [endless-sky/endless-sky#10701]
### Problem Description

Very simple request here: I'd like to be able to paste into the shipyard's name-field when buying a new ship

### Related Issue Links

None could be found

### Desired Solution

Allow pasting via CTRL+V into the ship name field when buying a new ship

### Alternative Approaches

.

**Comment by VaelophisNyx** [comment]
I did search for this but nothing came up

**Comment by TomGoodIdea** [comment]
Yeah it's a limitation of our dialog input fields in general.

**Comment by xobes** [comment]
> Supporting 'Paste' fully would raise expectations - to also support 'Copy', mid-string insert/delete...

Yeah... I was trying to use the Home button the other day to insert at the front.  Then I tried the mouse.  Then I tried the arrow keys.  Then I deleted everything and retyped it.

**Comment by VaelophisNyx** [comment]
I'd argue it's a slippery slope but it'd be sliding up so...worth it?

**Comment by TomGoodIdea** [comment]
https://github.com/endless-sky/endless-sky/commit/9a74def11465a4c3b8bba408c92a68b35d607faf
